### PR TITLE
Add setting active index after delete image from gallery

### PR DESCRIPTION
**Pull request by pkaczmarczy**
[Issue] https://github.com/Samsung/TAU/issues/178
[Problem] After deleting currently selected section
          no active index was set.
[Solution] Add setting active section after deleting picture

Signed-off-by: Pawel Kaczmarczyk <p.kaczmarczy@samsung.com>

### CODE REVIEW

**Comment by pkaczmarczy** [comment]
This PR is a result of cherry-picking commit which fixes Gallery page (image deleting issue)
Original PR: https://github.com/Samsung/TAU/pull/179

File: examples/mobile/UIComponents/components/gallery/gallery.js [hunk 1]
```diff
@@ -60,8 +60,12 @@
 		var activeIndex = sectionChangerWidget.getActiveSectionIndex();
 
 		sectionsParentNode.removeChild(sections[activeIndex]);
-		sectionChangerWidget.refresh();
+
 		resetChildSectionInfo();
+		if (sectionsLength > 0) {
+			sectionChangerWidget.setActiveSection((activeIndex - 1) >= 0 ? activeIndex - 1 : 0);
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/gallery/gallery.js]: simpler:
activeIndex > 0 ? activeIndex - 1 : 0;

Comment (pkaczmarczy) [reply examples/mobile/UIComponents/components/gallery/gallery.js]: This commit has been already merged. It's cherry-picked from tau_1.1 branch. I would prepare another commit  to 1.0 branch to be also merged into tau_1.1

Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/gallery/gallery.js]: ok. let's merge it.

**Pull request merged**